src/client/src/routes/StyleguideRoute/styled/rules.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from 'rt-theme'
+
+export { mapMarginPaddingProps, MarginPaddingProps } from './mapMarginPaddingProps'
+
+export const preventStutter = css`
+  -webkit-backface-visibility: hidden;
```

Comment:
I think that even though these properties are unsupported or experimental, the preprocessor will auto-prefix.